configs/openSUSE/dbus-services.toml

```diff
@@ -1,270 +1,750 @@
+bug = "bnc#742751"
+skips = [
+    "/usr/share/dbus-1/system-services/org.freedesktop.UDisks2.service",
+    "/usr/share/dbus-1/system.d/org.freedesktop.UDisks2.conf",
+]
+
+[[FileDigestGroup]]
+package = "scmon"
+type = "dbus"
+note = "legacy: not audited"
+skips = [
+    "/etc/dbus-1/system.d/com.novell.Pkcs11Monitor.conf",
+]
+
+[[FileDigestGroup]]
+package = "systemd"
+type = "dbus"
+note = "legacy: not audited"
+bug = "bnc#641924"
+skips = [
+    "/usr/share/dbus-1/services/org.freedesktop.systemd1.service",
+    "/usr/share/dbus-1/system-services/org.freedesktop.systemd1.service",
+    "/usr/share/dbus-1/system.d/org.freedesktop.systemd1.conf",
+    "/usr/lib/systemd/system/dbus-org.freedesktop.hostname1.service",
+    "/usr/share/dbus-1/system-services/org.freedesktop.hostname1.service",
+    "/usr/share/dbus-1/system.d/org.freedesktop.hostname1.conf",
+    "/usr/share/dbus-1/system.d/org.freedesktop.login1.conf",
+    "/usr/lib/systemd/system/dbus-org.freedesktop.login1.service",
+    "/usr/share/dbus-1/system-services/org.freedesktop.login1.service",
+    "/usr/share/dbus-1/system.d/org.freedesktop.timedate1.conf",
+    "/usr/lib/systemd/system/dbus-org.freedesktop.timedate1.service",
+    "/usr/share/dbus-1/system-services/org.freedesktop.timedate1.service",
+    "/usr/share/dbus-1/system.d/org.freedesktop.locale1.conf",
+    "/usr/lib/systemd/system/dbus-org.freedesktop.locale1.service",
+    "/usr/share/dbus-1/system-services/org.freedesktop.locale1.service",
+]
+
+# gconf2
+#"org.gnome.GConf.Defaults.service",
+#"org.gnome.GConf.Defaults.conf",
+
+# system-config-printer (bnc#694640)
+#"com.redhat.NewPrinterNotification.conf",
+#"com.redhat.PrinterDriversInstaller.conf",
+
+[[FileDigestGroup]]
+package = "rtkit"
+type = "dbus"
+note = "legacy: not audited"
+skips = [
+    "/usr/share/dbus-1/system.d/org.freedesktop.RealtimeKit1.conf",
+    "/usr/share/dbus-1/system-services/org.freedesktop.RealtimeKit1.service",
+]
+
+[[FileDigestGroup]]
+package = "wpa_supplicant"
+type = "dbus"
+note = "legacy: not audited"
+skips = [
+    "/usr/lib/systemd/system/dbus-fi.epitest.hostap.WPASupplicant.service",
+    "/usr/share/dbus-1/system-services/fi.epitest.hostap.WPASupplicant.service",
+]
+
+[[FileDigestGroup]]
+package = "wpa_supplicant"
+type = "dbus"
+note = "legacy: not audited"
+bug = "bnc#681116"
+skips = [
+    "/usr/lib/systemd/system/dbus-fi.w1.wpa_supplicant1.service",
+    "/usr/share/dbus-1/system-services/fi.w1.wpa_supplicant1.service",
+    "/etc/dbus-1/system.d/wpa_supplicant.conf",
+]
+
+[[FileDigestGroup]]
+package = "kdebase4-workspace"
+type = "dbus"
+note = "legacy: not audited"
+skips = [
+    "/usr/share/dbus-1/system-services/org.kde.kcontrol.kcmclock.service",
+    "/usr/share/dbus-1/system.d/org.kde.kcontrol.kcmclock.conf",
+]
+
+# kdebase4-workspace
+#"org.kde.fontinst.service",
+#"org.kde.fontinst.conf",
+#"org.kde.ksysguard.processlisthelper.service",
+#"org.kde.ksysguard.processlisthelper.conf",
```

Comment:
kde4 is gone from Factory so this should be fine

Reply:
Actually, no, these still exist in `plasma5-workspace` etc. Please check all commented out candidates using the `query_archives_db` tool. For erxample like this: `query_archives_db -p "%org.kde.ksysguard%" -f rpm,path`.